product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Postgres Distributed (PGD) presets on BigAnimal
+navTitle: PGD presets on BigAnimal
+deepToC: true
+---
+
+## Commit scope
+
+[Commit scopes](../../../pgd/latest/durability/commit-scopes/) in PGD is a set of rules that describes the behavior of the system as transactions are committed. Because they define how transactions are replicated across a distributed database, they impact consistency and reliability.
```

Comment:
```suggestion
[Commit scopes](../../../pgd/latest/durability/commit-scopes/) in PGD are a set of rules that describe the behavior of the system as transactions are committed. Because they define how transactions are replicated across a distributed database, they have an effect on consistency, durability and performance.
```